Main.sublime-menu

```diff
@@ -1,31 +1,33 @@
 [
-  { "id": "preferences",
+  {
     "children": [
-      { "caption": "Package Settings",
-        "mnemonic": "P",
-        "id": "package-settings",
```

Comment:
These are actually not optional since one package *has* to define them and if your package happens to be the only one in an ST instance they would not receive a caption from somewhere else.

Only the top-level `preferences` is optional because that is defined in the default menu file already.

Reply:
I've reverted the commit, and made another commit with the spacer removed.